src/transaction/log_global.c

```diff
@@ -143,5 +135,29 @@ log_global::get_log_prior_receiver ()
   assert (m_prior_recver != nullptr);
   return *m_prior_recver;
 }
+
+void
+log_global::update_ps_consensus_flushed_lsa ()
```

Comment:
Nit.
You changed the order of the functions - presumably - to reflect the order in the header. (this is how I understand you reason)
However, this is not a must because it adds unwanted noise in the diffs.
And some poor guy (maybe still one of us) coming back to this change in a year or two, will have some ugly words to say about the change trying to trace the modification causing a nasty bug. :)